src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.cs

```diff
@@ -198,7 +197,7 @@ public static class Avx
         public static Vector256<float> Set(float e7, float e6, float e5, float e4, float e3, float e2, float e1, float e0) { throw null; }
         public static Vector256<double> Set(double e3, double e2, double e1, double e0) { throw null; }
         public static Vector256<T> Set1<T>(T value) where T : struct { throw null; }
```

Comment:
Should this rather be called `SetOne` ? For consistency with `SetZero`, `Vector<T>.One`, `Vector<T>.Zero`.

Reply:
I thought that `Set1` should not be consistent with `SetZero` or `Vector<T>.One`.
- `SetZero` stands for "set all elements to value of zero";
- `Vector<T>.One` stands for "set all elements to value of one";
- `Set1` stands for "set all elements to ***one value of XX***";

But I know `Set1` is not a good name that just follows C++ Intel intrinsic naming. Do you have suggestions for a better name?

Reply:
Just Set or SetAll might make sense.

Reply:
> Just Set or SetAll might make sense.

We have `Set` for multi-value initialization. `SetAll` makes sense to me. Thank you.